README.md

```diff
@@ -2,35 +2,38 @@
 
 ![Image of brimcap analyze](https://github.com/brimdata/brimcap/raw/main/brimcap.gif)
 
-A command line utility for converting pcap data into the flexible, searchable
+A command line utility for converting packet captures into the flexible, searchable
```

Comment:
Maybe just make this pcaps instead of packet captures or pcap data? Not sure if there are other packet capture formats but we only support pcap and ng-pcap file formats.
```suggestion
A command line utility for converting pcaps into the flexible, searchable
```